retail/interactive-tutorial/RetailSearch.Samples/SearchWithQueryExpansionSample.cs

```diff
@@ -97,8 +97,8 @@ public static class SearchWithQueryExpansionTutorial
     [Runner.Attributes.Example]
     public static IEnumerable<SearchResponse> Search()
     {
-        var projectNumber = Environment.GetEnvironmentVariable("PROJECT_NUMBER");
+        var projectId = Environment.GetEnvironmentVariable("PROJECT_ID");
```

Comment:
This is the right environment variable name on the testing infrastructure.
```suggestion
        var projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
```

Reply:
Here and in all others.